cltk/prosody/latin/clausulae_analysis.py

```diff
@@ -18,32 +18,38 @@ def __init__(self):
         return
 
     @staticmethod
-    def clausulae_analysis(prosody):
+    def clausulae_analysis(prosody: List) -> Dict:
```

Comment:
prefer:
List[str]
Dict[str,int]
otherwise running mypy will likely complain.

Reply:
Makes sense, added better type hinting 👍 PyCharm actually has pretty good support for that